dev/com.ibm.ws.install/test/com/ibm/ws/install/internal/ArtifactDownloaderUtilsTest.java

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************************n * Copyright (c) 2024 IBM Corporation and others.n * All rights reserved. This program and the accompanying materialsn * are made available under the terms of the Eclipse Public License 2.0n * which accompanies this distribution, and is available atn * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0n *n * Contributors:n *     IBM Corporation - initial API and implementationn *******************************************************************************/
+package com.ibm.ws.install.internal;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+/**
+ *
+ */
+public class ArtifactDownloaderUtilsTest {
+
+    /**
+     * Test method for {@link com.ibm.ws.install.internal.ArtifactDownloaderUtils#checkResponseCode(int)}.
+     */
+    @Test
+    public void testCheckResponseCodeOK() {
```

Comment:
glad to see unit tests for this, in the future people will hesitate to make changes to strengthen 'checkResponseCode' now that we group 400+ in with 200 in the passing tests